website/docs/utilities/dialog-primitive/partials/code/how-to-use.md

```diff
@@ -34,3 +34,51 @@ The `DialogPrimitive` serves as the foundation for dialog derived components lik
   </:footer>
 </Hds::DialogPrimitive::Wrapper>
 ```
+
+### Header titleTag
+
+The `@titleTag` argument changes the HTML element that wraps the `DialogPrimitive::Header` tagline and "title" content. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if the `DialogPrimitive` is used as a Split Window, the value should be `"h2"`. 
+
+```handlebars
+<div class="doc-dialog-primitive-grid-layout">
+  <div class="doc-dialog-primitive-flex-layout">
+    <Hds::Text::Display @tag="h1" @size="500">Page with Split Window</Hds::Text::Display>
+    <div class="doc-dialog-primitive-content-placeholder">
+     <Hds::Text::Body>Main page content </Hds::Text::Body>
+    </div>
```

Comment:
This should use a `Doc::Placeholder` component. Then you wouldn't have to add custom HTML.